### ISSUE TITLE
Path bug + lintfile() should throw an error when a file is not found

### ISSUE THREAD

**Issue by lufecir** [wjohnsto/tsconfig-lint#1]
Currently if the lintfile() method does not find a file, this is silently ignored and it seems like the file contains 0 errors. This should not be the case. If a file is not found, an error should be thrown specifiying the failing file path.

You might wonder how it might not be able find a file, since the file list is build from a glob pattern. Well I think this is a bug (although maybe not an easy one to solve).

Suppose have you the following file structure:

```
package.json
src/
src/tsconfig.json
src/**/*.ts
```

package.json contains the following script:

```
"tslint": "tsconfig-lint --use-glob src"
```

tsconfig contains the following glob pattern:

```
"filesGlob": "**/*.ts",
```

this results in file entries like

```
    "files": [
        "./form/dropdown/scripts/dropdown.ts",
        "./form/modal-popup/scripts/modal-popup-controller.ts",
```

But the current working dir for tsconfig-lint is the root folder (where package.json is located) and not 'src' so none of the files is found by the linter.

I was able to solve this by putting tsconfig.json also in the root folder, but had to dig into the code to figure out why it was not working. So getting an error, specifying that a file was not found would make this more obvious.

A fix for the path issue would of course be better. The paths are now relative to the location of tsconfig.json and this works fine for tsc, but not for the linter (which should also use the location of tsconfig as CWD).

**Comment by wjohnsto** [comment]
The right solution is to fix the path issue, I will look into that. As for the file not found issue, that is a little bit more tricky to determine, but I'll see what I can do.

**Comment by wjohnsto** [comment]
This is in tslint-glob@0.6.0
